websites/D/Diziroll/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "erdem",
+      "id": "292692556809109534"
+    },
+    "url": ["diziroll.com", "www.diziroll.com"],
+    "description": {
+      "en": "Diziroll is a website that you can watch series.",
+      "tr": "Diziroll bir dizi izleme sitesidir."
+    },
+    "service": "Diziroll",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/hY9GNiw.png",
+    "thumbnail": "https://i.imgur.com/YsL5RgB.jpg",

```

Comment:
why the white border around the logo and its kinda blurry?

Reply:
i think, this is better ( https://github.com/PreMiD/Presences/pull/3220/commits/0999b0db4afc67557d8f3eb525ac0ffd5684fe88 )

Reply:
i changed the rpc's logo
![image](https://user-images.githubusercontent.com/41632465/110608699-88ce4480-819d-11eb-89f4-7924862a64b8.png)